### ISSUE TITLE
changelog v1.0.2

### ISSUE THREAD

**Issue by filipswit** [sinara-hw/Kasli-SOC#69]
- [ ] fixed https://github.com/sinara-hw/Kasli-SOC/issues/68
- [ ] fixed https://github.com/sinara-hw/Kasli-SOC/issues/67
- [ ] added 24AA025E48T-I/OT and  24AA025E48-I/SN on separate I2C bus https://github.com/sinara-hw/Kasli-SOC/issues/66
- [ ] added jumpers to BoM https://github.com/sinara-hw/Kasli-SOC/issues/54
- [ ] Added alternate LDOs - TPS74901RGWT
- [ ] Added alternate GPIO expanders - PCA9539APW

**Comment by gkasprow** [comment]
@filipswit did you consult moving the EEPROMs to separate I2C interface with software guys?

**Comment by filipswit** [comment]
no, but default EEPROM, IC5 is not moved.
We can't add aditional footprint - https://github.com/sinara-hw/Kasli/issues/97, because of conflict with SFP3 addres, so I made same solution as in Kasli v2.0.2

**Comment by kaolpr** [comment]
I believe there was some discussion on version naming convention and patch number (according to <major>.<minor>.<patch>) was to be reserved only for compatible component replacements. Here I can see some issue fixes.

If such discussion was not written (honestly, don't remember) maybe we should write down rules on that.

**Comment by filipswit** [comment]
@gkasprow and @kaolpr so should I rename it to v1.1?

**Comment by gkasprow** [comment]
yes, since it requires software changes, there must be new revision

**Comment by gkasprow** [comment]
The R110 and R100 could be replaced with E24 values 
![image](https://user-images.githubusercontent.com/4325054/203549145-fd0ff5c8-63bf-4ad2-94bd-b237c30a0d95.png)

**Comment by kaolpr** [comment]
> yes, since it requires software changes, there must be new revision

@gkasprow How does it require software changes? IO expander has different address / registry map?

**Comment by gkasprow** [comment]
Before EEPROM was connected to shared bus, now it has a dedicated FPGA pins.

**Comment by gkasprow** [comment]
@filipswit there is yet another [issue ](https://github.com/sinara-hw/EEM_FMC_Carrier/issues/46#issuecomment-1317083831)that needs correction 
The MCP23017 expander port 7 must be output. So we need to swap DISABLE with FAULT
I'd add a reistors for revision ID like in Kasli + one resistor for variant ID (a kind of expanders assembled) placed on the same sheet as I2C expanders

**Comment by filipswit** [comment]
>  decouple it from LDOs 1uF caps with a few tens Ohm resistor

You mean to place resistor (22R?) in series with 1uF at all LDOs Vbias inputs?
BTW. I can place any different LDO if you have any other cheap and small.

> The MCP23017 expander port 7 must be output. So we need to swap DISABLE with FAULT
![image](https://user-images.githubusercontent.com/24258965/203559255-c42e032a-6772-4961-ac15-843dca317c5f.png)

So it looks that IC24 is fine (these are test points) and in IC25 I should swap 'VUSB_PRESENT_N', with 'SFP_0.LED'?
![image](https://user-images.githubusercontent.com/24258965/203559865-eac6cb95-a24f-49d6-9c55-ef3a8ec512ce.png)
> I'd` add a reistors for revision ID like in Kasli + one resistor for variant ID (a kind of expanders assembled) placed on the same sheet as I2C expanders

I'll try to fit 3-4 rev ID bits and one variant ID resistor

**Comment by gkasprow** [comment]
> > decouple it from LDOs 1uF caps with a few tens Ohm resistor
> 
> You mean to place resistor (22R?) in series with 1uF at all LDOs Vbias inputs? BTW. I can place any different LDO if you have any other cheap and small.

just add a single 22Ohm (or similar already in the BOM) at the LDO output after the 100nF cap.

**Comment by gkasprow** [comment]
> So it looks that IC24 is fine (these are test points) and in IC25 I should swap 'VUSB_PRESENT_N', with 'SFP_0.LED'?

yep, my bad

**Comment by filipswit** [comment]
Just verified the DS for this LDO, so it looks 470nF is absolute minimum i this case:
![image](https://user-images.githubusercontent.com/24258965/203604965-e3a2f4b8-2ee4-4631-8c01-4e0e6f27e0c1.png)

Would it be good enough solution?
![image](https://user-images.githubusercontent.com/24258965/203605172-db56e9a8-cd2d-4c1a-88f3-2e0cb9097bf1.png)

**Comment by gkasprow** [comment]
OK, so you found better DS than me :)

**Comment by gkasprow** [comment]
No need for series resistor then.

**Comment by filipswit** [comment]
@gkasprow could you please look at the changes one more time?

**Comment by kaolpr** [comment]
@gkasprow @filipswit  Can we assume v1.1 is reviewed?

**Comment by kaolpr** [comment]
@gkasprow Do you think https://github.com/sinara-hw/Kasli/issues/99 should be implemneted here?

**Comment by gkasprow** [comment]
SOT-23 version of EEPROM is still not available

**Comment by gkasprow** [comment]
please add note close to the I2C expanders that ASSY variant need to be updated.

**Comment by gkasprow** [comment]
why both EEPROMs are not mounted?
![image](https://user-images.githubusercontent.com/4325054/210097594-1529c188-6465-4701-8838-2f39b37e5941.png)

**Comment by gkasprow** [comment]
Ah, I see, IC5 is connected instead...
I'd add 24AA025E48T-I/OT assembly variant to it as well.

**Comment by filipswit** [comment]
yeep, it should be added. I've made issue to not forget about it in next revision https://github.com/sinara-hw/Kasli-SOC/issues/72.
